src/css/custom.css

```diff
@@ -5,26 +5,94 @@
  */
 
 /* You can override the default Infima variables here. */
+
+@font-face {
+  font-family: "Inter";
+  src: url("/fonts/Inter-VariableFont_slnt,wght.ttf");
+}
+
+@font-face {
+  font-family: "Montserrat";
+  src: url("/fonts/Montserrat-VariableFont_wght.ttf");
+}
```

Comment:
Do we need to bundle the license if we are vendoring the assets?

Reply:
Not necessarily... But I'd at least add a note on the Readme where we list our license that we use these typefaces and a link to their licenses. 

Also is there a reason to use the ttf is it to avoid using Google fonts or performance wise

Reply:
> Not necessarily... But I'd at least add a note on the Readme where we list our license that we use these typefaces and a link to their licenses.
> 
> Also is there a reason to use the ttf is it to avoid using Google fonts or performance wise

Okay noted. 

I have downloaded ttf fonts from Google fonts and used the fonts locally, increases a little bit of performance. Good to have all basic stuff in root folder.

Reply:
I think this is still pending, right? Or did you address it? If so, mark as resolved.  Thanks! 🙏